.github/scripts/support/monday.js

```diff
@@ -546,6 +571,69 @@ module.exports = function Monday(issue) {
     return body?.match(mondayIdRegex)?.[0];
   }
 
+  /**
+   * Add or remove a label from a DropdownValues object. Creates a new object if none exists.
+   * @private
+   * @param {MondayLabel} labelInfo - The label information
+   * @param {("add" | "remove")} action - The action to perform: "add" or "remove"
+   * @returns {{ labels: string[] } | string} - The updated dropdown object or an empty string if no labels remain
+   */
+  function createDropdownValues(labelInfo, action) {
+    if (action !== "add" && action !== "remove") {
+      throw new Error(`Invalid action "${action}" in createDropdownValues. Use "add" or "remove".`);
+    }
+
+    const labelValue = String(labelInfo.value);
+    const currentValue = columnUpdates[labelInfo.column];
+    const existingLabels =
+      currentValue && typeof currentValue === "object" && "labels" in currentValue ? currentValue.labels : [];
+
+    if (existingLabels.length === 0 && labels?.length) {
+      for (const { name } of labels) {
+        const info = labelMap.get(name);
+        if (info?.column === labelInfo.column && info?.value) {
+          existingLabels.push(String(info.value));
+        }
+      }
+    }
+
+    const dropdownSet = new Set(existingLabels);
+    const present = dropdownSet.has(labelValue);
+    if (action === "add" && !present) {
+      dropdownSet.add(labelValue);
+    } else if (action === "remove" && present) {
+      dropdownSet.delete(labelValue);
+    }
+
+    return dropdownSet.size ? { labels: Array.from(dropdownSet) } : "";
+  }
+
+  /**
+   * Update a label in columnUpdates based on the action
+   * @private
+   * @param {string} label - The label name to update
+   * @param {("add" | "remove")} action - The action to perform
+   */
+  function updateLabel(label, action) {
+    if (!labelMap.has(label)) {
+      console.log(`Label "${label}" not found in Monday Labels map.`);
+      return;
+    }
+
+    const info = labelMap.get(label);
+    if (!info?.column || !info?.value) {
+      console.log(`Label "${label}" is missing column or title information.`);
+      return;
+    }
+
+    const isDropdown = info.column === columnIds.typeDropdown;
+    if (action === "add") {
+      setColumnValue(info.column, isDropdown ? createDropdownValues(info, "add") : info.value);
+    } else if (action === "remove" && info.clearable) {
```

Comment:
Isn't `action === "remove"` guaranteed here due to the first condition and supported `actions`?